src/lib.rs

```diff
@@ -14,147 +14,140 @@
 // limitations under the License.
 //!
 //! # Introduction
-//! The `geojson` crate reads and writes `GeoJSON` ([IETF RFC 7946](https://tools.ietf.org/html/rfc7946)) files,
-//! optionally using `serde` for serialisation. Crate users are encouraged to familiarise themselves with the spec,
-//! as the crate is structured around it.
-//! # Structure of the Crate
-//! GeoJSON can contain one of three top-level objects, reflected in the top-level `geojson::GeoJson`
-//! enum members of the same name:
-//!
-//! - [`Feature`](struct.Feature.html)
-//! - [`FeatureCollection`](struct.FeatureCollection.html)
-//! - [`Geometry`](struct.Geometry.html)
-//!
-//! With `FeatureCollection` being the most commonly used, since it can contain multiple child objects.
-//! A `FeatureCollection` contains `Feature` objects, each of which contains a `Geometry` object, which may be empty.
-//! Since the most common uses cases for GeoJSON involve the use of `Feature` and `FeatureCollection` objects,
-//! conversions to `Feature` are provided for both `Value` enum members and Geometry objects via the `From` trait.
-//! A potentially complicating factor is the `GeometryCollection` geometry type, which can contain
-//! one more `Geometry` objects, _including nested `GeometryCollection` objects_.
-//! The use of `GeometryCollection` is discouraged, however.
-//!
-//! If your primary use case for this crate is ingesting `GeoJSON` strings in order to process geometries
-//! using the algorithms in the [`geo`](https://docs.rs/geo) crate, you can do so by enabling the `geo-types` feature in
-//! your `Cargo.toml`, and using the [`quick_collection`](fn.quick_collection.html) function to
-//! parse [`GeoJson`](enum.GeoJson.html) objects into
-//! a [`geo_types::GeometryCollection`](../geo_types/struct.GeometryCollection.html).
-//! See [here](#conversion-to-geo-objects) for details.
-//!
-//! Conversely, if you wish to produce a `FeatureCollection` from a homogenous collection of `geo` types, a `From` impl is
-//! provided for `geo_types::GeometryCollection`:
 //!
-//! ```rust
-//! # #[cfg(feature = "geo-types")]
-//! // The geo-types feature is required for this functionality
-//! # #[cfg(feature = "geo-types")]
-//! use geojson::FeatureCollection;
-//! # #[cfg(feature = "geo-types")]
-//! use geo_types::{GeometryCollection, LineString, Polygon};
-//! # #[cfg(feature = "geo-types")]
-//! use std::iter::FromIterator;
+//! This crate helps you read and write [GeoJSON](https://geojson.org) — a format for encoding
+//! geographic data structures.
 //!
-//! # #[cfg(feature = "geo-types")]
-//! let poly = Polygon::new(
-//!     LineString::from(vec![(0., 0.), (1., 1.), (1., 0.), (0., 0.)]),
-//!     vec![],
-//! );
-//!
-//! # #[cfg(feature = "geo-types")]
-//! let polys = vec![poly];
-//! # #[cfg(feature = "geo-types")]
-//! let gc = GeometryCollection::from_iter(polys);
```

Comment:
ack, I lost this example. I'm going to re-add it. One minute!

Reply:
Ok, restored!

I did move it to the `### From geo-types to geojson` section below. It seemed a little out of place as the very first example in the doc.